vita3k/gui/src/perf_overlay.cpp

```diff
@@ -45,20 +45,22 @@ static ImVec2 get_perf_pos(ImVec2 window_size, EmuEnvState &emuenv) {
 
 static float get_perf_height(EmuEnvState &emuenv) {
     switch (emuenv.cfg.performance_overlay_detail) {
-    case MAXIMUM: return 138.f;
-    case MEDIUM: return 80.f;
+    case MAXIMUM: return 128.f;
+    case MEDIUM: return 70.f;
     case LOW:
     case MINIMUM:
     default: break;
     }
 
-    return 57.f;
+    return 47.f;
 }
 
 void draw_perf_overlay(GuiState &gui, EmuEnvState &emuenv) {
-    const auto MAIN_WINDOW_SIZE = ImVec2((emuenv.cfg.performance_overlay_detail == MINIMUM ? 95.5f : 152.f) * emuenv.dpi_scale, get_perf_height(emuenv) * emuenv.dpi_scale);
+    static const ImVec4 PERF_OVERLAY_BG_COLOR = ImVec4(0.282f, 0.239f, 0.545f, 0.8f);
+    const auto MAIN_WINDOW_SIZE = ImVec2((emuenv.cfg.performance_overlay_detail == MINIMUM ? 151.5f : 286.5f) * emuenv.dpi_scale, get_perf_height(emuenv) * emuenv.dpi_scale);
+    const auto WINDOW_SIZE = ImVec2((emuenv.cfg.performance_overlay_detail == MINIMUM ? 135.f : 270.f) * emuenv.dpi_scale, (emuenv.cfg.performance_overlay_detail <= LOW ? 35.f : 58.f) * emuenv.dpi_scale);
     const auto WINDOW_POS = get_perf_pos(MAIN_WINDOW_SIZE, emuenv);
-    const auto WINDOW_SIZE = ImVec2((emuenv.cfg.performance_overlay_detail == MINIMUM ? 72.5f : 130.f) * emuenv.dpi_scale, (emuenv.cfg.performance_overlay_detail <= LOW ? 35.f : 58.f) * emuenv.dpi_scale);
+    const auto SCALE_MAX = emuenv.max_fps < 300 ? emuenv.max_fps : 300;
```

Comment:
why block it on 300 ? digimon have issue and get 1200 fps or more in intro, is good for can detect any problem on vsync etc

Reply:
Digimon does not render at 1200 fps, it just how we do the computation that is wrong (Digimon renders with an immediate instead of waiting for the next vsync but that doesn't increase the fps). Like I said it's impossible for the PS Vita hardware to render anything at a fps above 60

Reply:
i know,  and no it really run to 1200, i have checked in other programe before, i have see also otheér game have strange big speed, but here block on 300 is no sense